resources/public/api/conf/1.0/examples/create_foreign_property_request.json

```diff
@@ -0,0 +1,52 @@
+{
+  "fromDate": "2018-04-06",
+  "toDate": "2019-04-06",
+  "foreignFhlEea": {
+    "income": {
+      "rentAmount": 5000.99,
+      "taxDeducted": 5000.99
+    },
+    "expenditure": {
+      "premisesRunningCosts": 5000.99,
+      "repairsAndMaintenance": 5000.99,
+      "financialCosts": 5000.99,
+      "professionalFees": 5000.99,
+      "costsOfServices": 5000.99,
+      "travelCosts": 5000.99,
+      "other": 5000.99,
+      "consolidatedExpenses": 5000.99
+    }
+  },
+  "foreignProperty": [
+    {
+      "foreignPropertyEntry": [
+        {
```

Comment:
You don't need to include foreignPropertyEntry in here.

It's mostly just the name of the array we use in the code, shouldn't be in the schema either